tests/phpunit/tests/blocks/editor.php

```diff
@@ -447,13 +447,19 @@ public function test_wp_get_post_content_block_attributes() {
 			),
 		);
 		// With no block theme, expect an empty array.
```

Comment:
```suggestion
		// With no block theme, expect null.
```
Update docblock based on changes.

Reply:
Oooh thanks, totally missed that 😄